files/image_config/monit/memory_checker

```diff
@@ -59,6 +69,81 @@ def get_command_result(command):
 
     return command_stdout.strip()
 
+def get_container_id(container_name):
+    """Gets full container ID of the specified container
+    Args:
+        container_name: A string indicates the name of specified container.
+    Returns:
+        container_id: A string indicates the full ID of specified container.
+    """
+    container_id = ""
+
+    get_container_info_cmd = "docker ps --no-trunc"
```

Comment:
>"docker ps --no-trunc" [](http://example.com/codeflow?start=29&length=22)

I recall that in the `get_command_result()` function, when using subprocess with shell=False, the command should be provided as a list of strings rather than a single string. Could you double-check this? Same for other places